docs/CODEOWNERS

```diff
@@ -38,6 +35,9 @@
 /client/sysinfo/ @koute
 /client/tracing/ @koute
 
+# FRAME
+/frame/ @paritytech/frame-coders
+
```

Comment:
Please add `paritytech/frame-coders` and `cheme` for `/frame/state-trie-migration/`.